### PR TITLE
Add starship and csv

### PR DESCRIPTION
**Pull request by lencx**
changelog: add starship and csv

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -42,6 +42,7 @@ I renamed the repository to "Awesome Alternatives in Rust". The original name wa
 #### bash/PowerShell/fish
 
 * [nushell](https://github.com/nushell/nushell/) - An attractive structured shell
+* [starship](https://github.com/starship/starship) - The minimal, blazing-fast, and infinitely customizable prompt for any shell!
```
Comment (TaKO8Ki) [inline README.md:45]: starship is not shell but cross-shell prompt. So, I think `bash/PowerShell/fish` subsection should not have starthip.

Comment (lencx) [reply README.md:45]: Sorry, I’m a bit confused about this category, I don’t know where it should be placed.

File: README.md [hunk 2]
```diff
@@ -161,6 +162,11 @@ I renamed the repository to "Awesome Alternatives in Rust". The original name wa
 
 * [Zellij](https://github.com/zellij-org/zellij) - A terminal workspace with batteries included
 
+#### csv
```
Comment (TaKO8Ki) [inline README.md:165]: This section should be original command line tool to which xsv and csview are alternatives. Is `csv` a command line tool? If so, could you add a repository or homepage url?

Comment (lencx) [reply README.md:165]: This is not the original command line tool, and I am not sure if it exists. So where should I put `csv`?

File: README.md (hunk 1 above)
Comment (TaKO8Ki) [reply README.md:45]: Hmm. I have no idea about good original item to which starship is an alternative. Considering this repository is a curated list of replacements for existing software written in Rust, I think starship should not be added even though starship is a great project.

File: README.md (hunk 2 above)
Comment (TaKO8Ki) [reply README.md:165]: If that's so, the two CLIs don't have an original CLI. So, I think they should not be added even though they are useful tools.

File: README.md (hunk 1 above)
Comment (lencx) [reply README.md:45]: respect your suggestion

File: README.md (hunk 2 above)
Comment (TaKO8Ki) [reply README.md:165]: I think they should be added to [awesome-rust](https://github.com/rust-unofficial/awesome-rust). xsv has already been added.

Comment (lencx) [reply README.md:165]: Okay, then I close it this pr.

Comment (TaKO8Ki) [reply README.md:165]: Thank you for trying to contribute to this repository.

**Pull request closed without merging**